apps/hub-app/src/components/DaoCard.tsx

```diff
@@ -129,7 +129,7 @@ export const DaoCard = ({
         )}
       </div>
       <div className="tag-box">
-        <Tag>{networkId}</Tag>
+        <Tag className="network">{networkId}</Tag>
```

Comment:
There is a Tag component in the component lib.
It might need a little refactor to align with the different types but shouldn't be too bad.

Currently the tag component is set up to do take the prop tagColor: 'blue' | 'green' | 'pink' | 'violet';
It would be better to define the specific variants of a tag and pass those as props (ex. 'network' | 'delegate' | etc...). Then you can apply those either dynamically or create class names out of the prop.